### DISCUSSION TITLE
IconButton with Text?

### DISCUSSION THREAD

**Discussion by eric-hc** [chakra-ui/chakra-ui#2692] [Q&A] [answered]
Is it possible to create an IconButton that uses [a custom Icon](https://chakra-ui.com/docs/components/icon#using-the-icon-component) and Text? 

Similar to these: https://moderncss.dev/icon-button-css-styling-guide/ & https://www.w3schools.com/howto/howto_css_icon_buttons.asp

**Comment by with-heart** [comment, marked as answer]
Hi @eric-hc!

Icon only button: https://chakra-ui.com/docs/form/icon-button
Text+icon button: https://chakra-ui.com/docs/form/button#button-with-icon

**Reply by eric-hc** [reply]
Duh! Totally didn't even think about it that way. Thanks!

**Comment by voyagebagage** [comment]
and a simple icon with text?
